build.py

```diff
@@ -44,33 +44,43 @@ def build_mac(release):
         return False
     return True
 
+def check_pkgconfig(lib, min_version):
+    try:
+        subprocess.check_output(
+            ['pkg-config', '--exists', '%s >= %s' % (lib, min_version)])
```

Comment:
I was building against harfbuzz 1.3.2 (as shipped by fedora) and it seems to work.